x-pack/plugins/security_solution/public/timelines/components/timeline/body/actions/index.tsx

```diff
@@ -128,6 +130,24 @@ const ActionsComponent: React.FC<ActionProps> = ({
     }
   }, [dispatch, ecsData._id, timelineId, setGlobalFullScreen, setTimelineFullScreen]);
 
+  const entryLeader = useMemo(() => {
+    const { process } = ecsData;
+    const entryLeaderIds = process?.entry_leader?.entity_id;
+    if (entryLeaderIds !== undefined) {
+      return entryLeaderIds[0];
```

Comment:
Always going to be the first one? And it's guaranteed to be an array of 1?

Reply:
We need to check array length at least.